python/py-pytest-benchmark/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytest-benchmark
+version             3.2.3
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         A pytest fixture for benchmarking code.
+
+long_description    {*}${description} It will group the tests into \
+                    rounds that are calibrated to the chosen timer.
+
+homepage            https://github.com/ionelmc/pytest-benchmark
+
+checksums           rmd160  c92e3b9508d59c73c723aebcf3f8a783967bba16 \
+                    sha256  ad4314d093a3089701b24c80a05121994c7765ce373478c8f4ba8d23c9ba9528 \
+                    size    337882
+
+supported_archs     noarch
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-cpuinfo \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-tox
```

Comment:
I would be surprised to have `tox` as a runtime dependency for a pytest plugin... are you sure?

Reply:
Thanks—cut-and-paste error fixed.